<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>principale_ihm</class>
 <widget class="QMainWindow" name="principale_ihm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Royaume de Siam</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Téléchargements/logo.png</normaloff>../Téléchargements/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="conteneur" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>561</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bouton_rec">
       <property name="text">
        <string>Restart</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_pla">
       <property name="text">
        <string>Placer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_dep">
       <property name="text">
        <string>Deplacer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_ori">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Réorienter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_sor">
       <property name="text">
        <string>Sortir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_aid">
       <property name="text">
        <string>Aide</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_mus">
       <property name="text">
        <string>Musique</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichiers">
    <property name="title">
     <string>Fichiers</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichiers"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>principale_ihm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
